docs/literate/src/files/getting_started_with_Trixi.jl

```diff
@@ -0,0 +1,287 @@
+#src # Getting started with Trixi.jl
+
+# This tutorial is intended for beginners in Julia and Trixi.jl.
+# After reading it, you will install Julia and Trixi on your PC and will be able to download files
+# from Trixi github, execute and edit them.
+# **Trixi.jl** is a numerical simulation framework for hyperbolic conservation laws 
+# written in [`Julia`](https://julialang.org/).
+# This means that Julia have to be installed on a PC to work with Trixi. 
+
+# ## Julia installation
+
+# Trixi works with the current stable Julia release. More information about Julia support can be
+# found in [`README`](https://github.com/trixi-framework/Trixi.jl#installation).
+# The most fully explained installation process can be found in
+# [`Julia installation instruction`](https://julialang.org/downloads/platform/).
+# But you can follow also our short installation instruction.
+
+# ### Windows
+
+# - Download Julia [`installer`](https://julialang.org/downloads/) for Windows. Make sure 
+#   that you choose the right version of installer (64-bit or 32-bit) according to your computer.
+# - Open the downloaded installer.
+# - Paste an installation directory path or find it using a file manager (select `Browse`).
+# - Select `Next`.
+# - Check the `Add Julia to PATH` to add Julia to the Environment Variables. 
+#   This makes possible to run Julia using Terminal from any directory only typing `julia`.
+# - Select `Next`, then Julia will be insalled.
+
+# Now you can verify, that Julia is installed:
+# - Type `Win+R` on a keyboard.
+# - Enter `cmd` in opened window.
+# - Enter in a terminal `julia`. 
+
+# Then Julia will be invoked. To close Julia enter `exit()`. 
+
+# ### Linux
+
+# - Open a Terminal and navigate (using `cd`) to the directory, where you want to save Julia.
+#   Or you can open a file manager, find this directory, right-click inside and 
+#   choose `Open Terminal Here`.
+# - To install Julia execute the following commands in a Terminal:
+#   ````
+#   wget https://julialang-s3.julialang.org/bin/linux/x64/1.8/julia-1.8.5-linux-x86_64.tar.gz
+#   tar zxvf julia-1.8.5-linux-x86_64.tar.gz
+#   ````
+#   Now you can verify that Julia is installed entering `julia` command in a Terminal.
+
+# Then Julia will be invoked. To close Julia enter `exit()`.
+
+# ## Trixi installation
+
+# Trixi and its related tools are registered Julia packages. So installation of them is
+# running inside Julia. For appropriate work of Trixi you need to install 
+# [`Trixi`](https://github.com/trixi-framework/Trixi.jl),
+# [`Trixi2Vtk`](https://github.com/trixi-framework/Trixi2Vtk.jl),
+# [`OrdinaryDiffEq`](https://github.com/SciML/OrdinaryDiffEq.jl) and 
+# [`Plots`](https://github.com/JuliaPlots/Plots.jl).
```

Comment:
```suggestion
# Trixi and its related tools are registered Julia packages, thus their installation
# happens inside Julia.
# For a smooth workflow experience with Trixi.jl, you need to install 
# [Trixi.jl](https://github.com/trixi-framework/Trixi.jl),
# [OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl) and 
# [Plots.jl](https://github.com/JuliaPlots/Plots.jl).
```
Some rewording. I also removed Trixi2Vtk, since ParaView is not mentioned in this tutorial at all, so it is not strictly necessary here I believe.